io/disk/iozone.py

```diff
@@ -0,0 +1,577 @@
+                       'F READ', 'FRE READ']
+        logging.info("\n%s", astring.tabular_output(
+            record_size, header=header_list))
+
+        logging.info("ANALYSIS of DRILLED DATA:")
+
+        logging.info("")
+        logging.info("TABLE:  RECsize Difference between runs                 "
+                     "Results are % DIFF")
+        logging.info("")
+        logging.info("REGRESSIONS: %d (%.2f%%)    Improvements: %d (%.2f%%)",
+                     record_regressions,
+                     (100 * record_regressions / float(record_total)),
+                     record_improvements,
+                     (100 * record_improvements / float(record_total)))
+        logging.info("")
+
+        logging.info("")
+        logging.info("TABLE:  FILEsize Difference between runs               "
+                     "Results are % DIFF")
+        logging.info("")
+
+        logging.info("\n%s", astring.tabular_output(
+            file_size, header=header_list))
+        logging.info("REGRESSIONS: %d (%.2f%%)    Improvements: %d (%.2f%%)",
+                     file_regressions,
+                     (100 * file_regressions / float(file_total)),
+                     file_improvements,
+                     (100 * file_improvements / float(file_total)))
+        logging.info("")
+
+    def analyze(self):
+        """
+        Analyzes and eventually compares sets of IOzone data.
+        """
+        overall = []
+        record_size = []
+        file_size = []
+        for path in self.list_files:
+            c_file = open(path, 'r')
+            logging.info('FILE: %s', path)
+
+            results = self.parse_file(c_file)
+
+            overall_results = self.process_results(results)
+            record_size_results = self.process_results(results, 'record_size')
+            file_size_results = self.process_results(results, 'file_size')
+            self.report(overall_results, record_size_results,
+                        file_size_results)
+
+            if len(self.list_files) == 2:
+                overall.append(overall_results)
+                record_size.append(record_size_results)
+                file_size.append(file_size_results)
+
+        if len(self.list_files) == 2:
+            record_comparison = data_structures.compare_matrices(*record_size)
+            file_comparison = data_structures.compare_matrices(*file_size)
+            self.report_comparison(record_comparison, file_comparison)
+
+
+class IOzonePlotter(object):
+
+    """
+    Plots graphs based on the results of an IOzone run.
+
+    Plots graphs based on the results of an IOzone run. Uses gnuplot to
+    generate the graphs.
+    """
+
+    def __init__(self, results_file, output_dir):
+        self.active = True
+        s_mg = SoftwareManager()
+        if not s_mg.check_installed("gnuplot") and not s_mg.install("gnuplot"):
+            logging.error("Command gnuplot not found, disabling graph "
```

Comment:
and also here, basically you should not use `logging` directly, unless necessary.